src/main/java/eu/darkbot/api/managers/StarAPI.java

```diff
@@ -0,0 +1,82 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.API;
+import eu.darkbot.api.entities.utils.Area;
+import eu.darkbot.api.entities.utils.Map;
+import eu.darkbot.api.utils.Listener;
+import eu.darkbot.utils.ArrayUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * API to add, manage, retrieve maps.
+ */
+public interface StarAPI extends API {
+    List<String> HOME_MAPS = ArrayUtils.asImmutableList("1-1", "2-1", "3-1");
+    List<String> OUTPOST_HOME_MAPS = ArrayUtils.asImmutableList("1-8", "2-8", "3-8");
+
+    /**
+     * @return current {@link Map}
+     */
+    Map getCurrentMap();
+
+    /**
+     * @return bounds of the current map
+     */
+    Area.Rectangle getCurrentMapBounds();
+
+    /**
+     * Adds given map into list of maps.
+     *
+     * @return true if was added, false when list already contains given {@link Map}
+     */
+    boolean addMap(@NotNull Map map);
```

Comment:
I'm not convinced that one should just be causally "adding" maps via API, if anything it should be a `getOrCreateMapById` where if a map doesn't exist it will just create a dummy map with that id and store it. It will also save you the allocaton of a new map that would be discarded if you can't add it